CRM/Contact/Page/View/Log.php

```diff
@@ -47,11 +48,15 @@ public function browse() {
 
     $logEntries = [];
     while ($log->fetch()) {
-      [$displayName, $contactImage] = CRM_Contact_BAO_Contact::getDisplayAndImage($log->modified_id);
+      if ($log->modified_id && !isset($modifiers[$log->modified_id])) {
+        $displayInfo = CRM_Contact_BAO_Contact::getDisplayAndImage($log->modified_id);
+        $modifiers[$log->modified_id] = ['name' => $displayInfo[0] ?? '', 'image' => $displayInfo[1] ?? ''];
+      }
+
       $logEntries[] = [
         'id' => $log->modified_id,
-        'name' => $displayName,
-        'image' => $contactImage,
+        'name' => $log->modified_id && $modifiers[$log->modified_id]['name'] ? $modifiers[$log->modified_id]['name'] : '',
```

Comment:
I think this should be better done as `$modifiers[$log->modified_id]['name'] ?? ''` right @eileenmcnaughton ?

Reply:
I agree. @shahrukh-compuco can you make that change please?

Reply:
@seamuslee001 @colemanw thank you for reviewing, I have changed this as per suggestion